src/thriveopedia/fossilisation/FossilisationDialog.tscn

```diff
@@ -51,7 +47,10 @@ margin_right = 152.0
 [node name="Label" parent="VBoxContainer" index="0"]
 margin_right = 137.0
 
-[node name="HBoxContainer" parent="VBoxContainer" index="1"]
+[node name="FocusGrabber" parent="VBoxContainer" index="1"]
```

Comment:
Why is there an unconfigured focus grabber node here?

Reply:
For some reason I don't know (Maybe @Oliveriver knows?) `FossilisationDialog` is extended from `CustomConfirmationDialog`. Any changes in the base scene will be added automatically here.

Reply:
I see you configured the focus grabber in a commit, I'll test this now.

Reply:
> For some reason I don't know (Maybe @Oliveriver knows?) `FossilisationDialog` is extended from `CustomConfirmationDialog`. Any changes in the base scene will be added automatically here.

Whether it's significant or not, I can't remember why this changed.

Reply:
Well I definitely added the focus grabber to the base confirmation dialog so that the cancel button is selected by default.

I commented elsewhere already but it seems the fossilisation dialog entirely just hides the usual confirmation dialog content, so it inheriting that, doesn't make much sense so I'm open to leaving things be as they are in this PR but opening a follow up issue to fix that.